docs/Changelog.md

```diff
@@ -2,6 +2,7 @@
 
 1.12.0 (Not released yet)
 -------------------------
+* [#4677](https://github.com/TouK/nussknacker/pull/4677) Added validation to SPEL string conversions (allow only constant that convert successfully)
```

Comment:
Please use `SpEL` notation

Reply:
`SpEL` not `SPeL`. It is from *Sp*ring *E*xpression *L*anguage

Reply:
:face_in_clouds: